assignments/callbacks.js

```diff
@@ -2,29 +2,42 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
+  cb(arr[0])
 }
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  cb(arr.length)
 }
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  cb(arr.length - 1)
```

Comment:
should pass the last item, not the index of the last item, so `cb(arr[arr.length-1])`